pallets/base/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "pallet-base"
```

Comment:
I'm tempted to call this pallet `check` but a agree with the choice of `base` if there is a forward thinking plan of perhaps moving some base methods from identity here.

Reply:
See above. :)